### PR TITLE
Welcome banner

### PR DESCRIPTION
**Pull request by rjstires**
image

### CODE REVIEW

File: src/WelcomeBanner.tsx [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+        <Grid container className={classes.content}>
+          <Grid item xs={12}>
+            <Typography>We've added new features to help improve your experience and create a fresh new look. Help us build a better product for Linode customers like you. Share you feedback in the footer below.</Typography>
+          </Grid>
+          <Grid item xs={12}>
+            <Grid container>
+              <Grid item xs={12} md={4} className={classes.item}>
+                <Streamline className={classes.icon} />
+                <Typography variant="subheading" className={classes.itemTitle}>Streamline Deployments</Typography>
+                <Typography variant="caption" className={classes.itemDesc}>Deploy a Linode, NodeBalancers, Block Storage Volume, or Domain easily with step-by-step guidance.</Typography>
```
Comment (martinmckenna) [inline src/WelcomeBanner.tsx]: Nodebalancers should be singular to match the others

File: src/WelcomeBanner.tsx [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+    return (
+      <ConfirmationDialog
+        open={this.props.open}
+        title="Welcome to the New Cloud Manager!"
+        className={classes.dialog}
+      >
+        <Grid container className={classes.content}>
+          <Grid item xs={12}>
+            <Typography>We've added new features to help improve your experience and create a fresh new look. Help us build a better product for Linode customers like you. Share you feedback in the footer below.</Typography>
```
Comment (Jskobos) [inline src/WelcomeBanner.tsx]: your

File: src/WelcomeBanner.tsx [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+              </Grid>
+              <Grid item xs={12} md={4} className={classes.item}>
+                <Resource className={classes.icon} />
+                <Typography variant="subheading" className={classes.itemTitle}>Find Resources</Typography>
+                <Typography variant="caption" className={classes.itemDesc}>Our new predictive search gives you quick access to support documentation and community posts.</Typography>
+              </Grid>
+              <Grid item xs={12} md={4} className={classes.item}>
+                <Mobile className={classes.icon} />
+                <Typography variant="subheading" className={classes.itemTitle}>Access Anywhere</Typography>
+                <Typography variant="caption" className={classes.itemDesc}>With support for all major devices and screen sizes, it's easy to stay connected</Typography>
```
Comment (Jskobos) [inline src/WelcomeBanner.tsx]: period

**Comment by patthiel** [comment]
Also, the CI test failure is due to a flakey functional test, I'm pushing up a PR to address it now.

**Pull request merged**